balena.yml

```diff
@@ -9,7 +9,7 @@ post-provisioning: >-
   
   When it's done you should be able to access the access the app at http://pihole.local with a default password of `balena`.
 
-  On your router or DHCP server assign a static IP to your Pi-hole device, and set your clients DNS to the same IP address.
+  On your router or DHCP server assign a static IP resreservation (if possible) to your Pi-hole device, and set your clients DNS to the same IP address.
```

Comment:
```suggestion
  On your router or DHCP server assign a static IP reservation (if possible) to your Pi-hole device, and set your clients DNS to the same IP address.
```

Reply:
I need to not submit changes while I'm playing Rocket League.  I'll clean these up in a few after I remember how to rebase.